recipes/xorg-proto/all/conanfile.py

```diff
@@ -0,0 +1,86 @@
+from conans import AutoToolsBuildEnvironment, ConanFile, tools
+import glob
+import os
+import re
+import yaml
+
+required_conan_version = ">=1.33.0"
+
+
+class XorgProtoConan(ConanFile):
+    name = "xorg-proto"
+    description = "This package provides the headers and specification documents defining " \
+        "the core protocol and (many) extensions for the X Window System."
+    topics = ("conan", "xproto", "header", "specification")
+    license = "X11"
+    homepage = "https://gitlab.freedesktop.org/xorg/proto/xorgproto"
+    url = "https://github.com/conan-io/conan-center-index"
+    settings = "os", "arch", "compiler", "build_type"
+
+    generators = "pkg_config"
+
+    _autotools = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _settings_build(self):
+        return self.settings_build if hasattr(self, "settings_build") else self.settings
+
+    def build_requirements(self):
+        self.build_requires("xorg-macros/1.19.3")
+        self.build_requires("pkgconf/1.7.4")
+
+    def package_id(self):
+        self.info.header_only()
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version],
+                  destination=self._source_subfolder, strip_root=True)
+
+    def _configure_autotools(self):
+        if self._autotools:
+            return self._autotools
+        self._autotools = AutoToolsBuildEnvironment(self, win_bash=self._settings_build.os == "Windows")
+        self._autotools.configure(configure_dir=self._source_subfolder)
+        return self._autotools
+
+    def build(self):
+        for patch in self.conan_data.get("patches", {}).get(self.version, []):
+            tools.patch(**patch)
```

Comment:
I don't see patches, but it doesn't hurt

Reply:
This is a restover from recipe development.
But it doesn't hurt keeping it here, indeed.